ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, seed, brain, trainer_parameters, load=False):
         if self.use_continuous_act:
             self.num_branches = self.brain.vector_action_space_size[0]
         self.model_path = trainer_parameters["model_path"]
+        self.initialize_ckpt_path = trainer_parameters.get("init_path", None)
```

Comment:
That's true! Would be kind of nice to be able to do that. The main reason I didn't was that currently the trainer_config.yaml has no directory-specific information, i.e. it will always run regardless of what's on your hard drive/summaries folder. 

What about providing it as an optional? I.e. you can still use the `--initialize-from` and it will supersede anything else, but if you specify in the `trainer_config.yaml` and don't use `--initialize-from` it will work as well.

Reply:
Well, demos are the current exception :-).  I don't think it's really important to add this in this PR, but that solution sounds reasonable to me.

Reply:
It actually already works :P I made it explicit in the documentation.